### ISSUE TITLE
"Start tracking": adding and stopping current activity in 2.0 is not obvious

### ISSUE THREAD

**Issue by axelsimon** [projecthamster/hamster#310]
Hello,

First of all, thanks for making Hamster, I've been using it for many years now and it has been very useful to me :)

Now, I may be missing something obvious, but since Hamster got updated to v2 (hamster-time-tracker-2.0-0.3.rc1.fc23.noarch on Fedora 23) I can't easily start an activity.

I have to click on the "+" button whereas I used to just be able to click the "start tracking button":
![old start tracking behaviour](https://c1.staticflickr.com/3/2715/4332774392_1302c2922e_o.png)

More annoying, I can't stop the current activity by just clicking "stop" (or maybe I'm just not finding how to). So I have to double-click the currently tracking activity to open the "edit details" dialog, then enter the current time next to the start-up time (as in: 18:00-18:15 reporting bug@Hamster) and save, to stop an activity. This significantly slows down the workflow I find.

Am I missing something?

Thanks

axel

**Comment by axmachado** [comment]
Hello, axel...

I'm missing the old UI too... the new one is not so productive. @FelixSchwarz is starting to work to port the old interface to GTK3 to use with hamster2.

Until then, I've discovered that the stop activity can be achieved using the CLI interface (hamster stop), but the restart with double click is really missing.

**Comment by Osndok** [comment]
LOL.... I *STILL* don't know how to start/stop/use the new UI... I had to dig up an old v1 rpm for personal use, and started a v1 fork so I can track v1 bugs, like this one: https://github.com/Osndok/hamster-v1/issues/1

**Comment by fralix** [comment]
Yeah, old UI was way more useful. I also miss the overview, the new one doesn't show the hour time.
